lighthouse/src/main.rs

```diff
@@ -324,6 +324,30 @@ fn main() {
                 .takes_value(true)
                 .global(true)
         )
+        .arg(
+            Arg::with_name("genesis-state-url")
+                .long("genesis-state-url")
+                .value_name("URL")
+                .help(
+                    "A URL of a beacon-API compatible server from which to download the genesis state. \
+                    Checkpoint sync server URLs can generally be used with this flag. \
+                    If not supplied, a default URL may be used. If the genesis state is already included \
```

Comment:
Perhaps worth mentioning that checkpoint-sync-url will be used if this is not supplied?

Reply:
Thanks, fixed in d18edb7c5